src/server/api/router.js

```diff
@@ -1,551 +0,0 @@
-    api.get(
-        "/apps/:systemId/:appId/details",
-        auth.authnTokenMiddleware,
-        terrainHandler({
-            method: "GET",
-            pathname: "/apps/:systemId/:appId/details",
-        })
-    );
-
-    logger.info("adding the PUT /apps/:systemId/:appId/favorite handler");
-    api.put(
-        "/apps/:systemId/:appId/favorite",
-        auth.authnTokenMiddleware,
-        terrainHandler({
-            method: "PUT",
-            pathname: "/apps/:systemId/:appId/favorite",
-        })
-    );
-
-    logger.info("adding the DELETE /apps/:systemId/:appId/favorite handler");
-    api.delete(
-        "/apps/:systemId/:appId/favorite",
-        auth.authnTokenMiddleware,
-        terrainHandler({
-            method: "DELETE",
-            pathname: "/apps/:systemId/:appId/favorite",
-        })
-    );
-
-    logger.info("adding the POST /apps/:systemId/:appId/rating handler");
-    api.post(
-        "/apps/:systemId/:appId/rating",
-        auth.authnTokenMiddleware,
-        terrainHandler({
-            method: "POST",
-            pathname: "/apps/:systemId/:appId/rating",
-            headers: {
-                "Content-Type": "application/json",
-            },
-        })
-    );
-
-    logger.info("adding the DELETE /apps/:systemId/:appId/rating handler");
-    api.delete(
-        "/apps/:systemId/:appId/rating",
-        auth.authnTokenMiddleware,
-        terrainHandler({
-            method: "DELETE",
-            pathname: "/apps/:systemId/:appId/rating",
-        })
-    );
-
-    logger.info("adding the GET /analyses handler");
-    api.get(
-        "/analyses",
-        auth.authnTokenMiddleware,
-        terrainHandler({
-            method: "GET",
-            pathname: "/analyses",
-        })
-    );
-
-    logger.info("adding the POST /analyses handler");
-    api.post(
-        "/analyses",
-        auth.authnTokenMiddleware,
-        terrainHandler({
-            method: "POST",
-            pathname: "/analyses",
-            headers: {
-                "Content-Type": "application/json",
-            },
-        })
-    );
-
-    logger.info("adding /analyses/:id/history");
-    api.get(
-        "/analyses/:id/history",
-        auth.authnTokenMiddleware,
-        terrainHandler({
-            method: "GET",
-            pathname: "/analyses/:id/history",
-        })
-    );
-
-    logger.info("adding /analyses/:id/parameters");
-    api.get(
-        "/analyses/:id/parameters",
-        auth.authnTokenMiddleware,
-        terrainHandler({
-            method: "GET",
-            pathname: "/analyses/:id/parameters",
-        })
-    );
-
-    logger.info("adding the POST /quicklaunches handler");
-    api.post(
-        "/quicklaunches",
-        auth.authnTokenMiddleware,
-        terrainHandler({
-            method: "POST",
-            pathname: "/quicklaunches",
-            headers: {
-                "Content-Type": "application/json",
-            },
-        })
-    );
```

Comment:
Oops, missed this in my review, but it looks like this endpoint wasn't ported into a new file, so it's missing now.

Reply:
😖